gen3_tracker/common/__init__.py

```diff
@@ -319,9 +320,42 @@ def create_id(resource, project_id) -> str:
     from gen3_tracker import ACED_NAMESPACE
     assert resource, "resource required"
     assert project_id, "project_id required"
-    # identifier string is not unique when it complies with FHIR server. Using something else
-    # identifier_string = identifier_to_string(resource.identifier)
-    return str(uuid.uuid5(ACED_NAMESPACE, f"{project_id}/{resource.resource_type}/{resource}"))
+    identifier_string = identifier_to_string(resource.identifier)
+    return str(uuid.uuid5(ACED_NAMESPACE, f"{project_id}/{resource.resource_type}/{identifier_string}"))
+
+
+def create_object_id(path: str, project_id: str) -> str:
+    """Create a unique did for this object within a project."""
```

Comment:
It took me a second to figure out the difference between `create_object_id` and `create_id`... would it be possible to clarify that the former is for document references only and the latter is for the rest of the resource types?

(maybe `create_id` renamed to `create_resource_id` and add to `create_object_id` that this is only for DocRefs?)

Reply:
done.